config/connectors.yml.example

```diff
@@ -1,5 +1,5 @@
 # general metadata
-version: "8.3.0-SNAPSHOT"
+version: "${VERSION}-${TIMESTAMP}"
```

Comment:
I just don't get it where are those values from? Will the `version` be set by 
https://github.com/elastic/connectors/blob/818872f5913abbfee249074c7f7657ba39debc70/Makefile#L25

and 

https://github.com/elastic/connectors/blob/818872f5913abbfee249074c7f7657ba39debc70/Makefile#L30

Reply:
the `version` value is overwritten by the makefile. we have one for dev previews (generated by the version.sh script)

Reply:
Yes, so where is `${VERSION}-${TIMESTAMP}` from? envVar?

Reply:
That's never used in practice -- could be CHANGEME

Reply:
I think I like CHANGEME or any other placeholder that does not use ${ENVIRONMENT_VARIABLES} to set expectations that we're not compiling the template version, but straight overwrite it

Reply:
Addressed in https://github.com/elastic/connectors/pull/72/commits/97a40b2af4876f12f9cf961aa1edf06348f9bb1b